[WIP] Updating napari's python and Bundled App Installation Instructions

dgmccart opened the pull request:
# Description
<!-- What does this pull request (PR) do? Why is it necessary? -->
<!-- Tell us about your new feature, improvement, or fix! -->
<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousand words!" -->
<!-- You can use https://www.cockos.com/licecap/ or similar to create animations -->
Fixing the broken link that points to the install as a bundled app section
Adding new images to the repository for the new content
Adding new verbiage for the bundled app installation copy

## Type of change
<!-- Please delete options that are not relevant. -->
- [X] Bug-fix (non-breaking change which fixes an issue)

## Final checklist:
- [X] My PR is the minimum possible work for the desired functionality

dgmccart committed:
Fixing broken link

Fixing the broken link that points to the install as a bundled app section

dgmccart committed:
Added files to images for documentation

Adding images for the new bundled app installation instructions

dgmccart committed:
Bundled app installation guide for new release

Updating the bundled app installation instructions for MacOS, Windows, and Linux based on the new instructions for the latest bundled app release

jaimergp commented on docs/tutorials/fundamentals/installation.md:
````diff
@@ -169,119 +166,142 @@ you're not using.
+![image: napari icon in MacOS laundpad](.../docs/images/bundle_13.png)
+
+```{note} 
+The first time you open napari you must use the Launchpad, but subsequently, the napari application will show up in Spotlight search.
````
```suggestion
The first time you open napari you must use the Launchpad, but subsequently, the napari application should show up in Spotlight search.
```

I am not sure if it happens immediately after one usage, or if it's more about macOS learning about your preferences (e.g. it requires more than one click to appear in Spotlight).

jaimergp commented on docs/tutorials/fundamentals/installation.md:
```diff
@@ -169,119 +166,142 @@ you're not using.
+![image: napari License Agreement](.../docs/images/bundle_18.png)
 
-The napari setup wizard will then open. Click "Next" to begin the installation.
+The recommended installation method is to install napari just for the current user. However, you may install for all users using administrator privileges.
```
A reminder that we haven't tested the All Users installation too much.

jaimergp commented on docs/tutorials/fundamentals/installation.md:
```diff
@@ -169,119 +166,142 @@ you're not using.
 
-![image: Windows napari setup wizard
-finished](../assets/tutorials/installation/windows_bundle_installer_finish.png)
+On the next page, we recommend you check ‘Clear the package cache upon completion’ since this frees up memory in your machine’s cache without compromising napari functionality following installation. Click ‘Install’ to start the installation process.
```
I'd avoid mentioning the "Clear the package cache" explicitly here. It's now available (and it's safe to use), but we might change that possibility in the future to deal with reinstallations, updates and other features.

```suggestion
On the next page, click ‘Install’ to start the installation process.
```

chili-chiu committed:
split Python package and bundled app installation

dgmccart committed:
Editing the copy of the bundle installation guide

Adding in @jaimergp suggestions and some other copy changes to the bundle installation guide

jni reviewed (changes requested):
Thanks @dgmccart!

Unfortunately our docs contribution process is a bit in-flux, but in no case do we want to merge documentation images into the repo, as this bloats the repo forever (because they will be in the history even if we remove them later). Currently, they need to be added to git-lfs, using something along the lines of:

```
brew install git-lfs
git lfs install
git add docs/images/*.png
```

I don't know how to move files already committed to lfs, maybe @melissawm has suggestions there.

Regarding the bundle install instructions, I thought we had decided to hold off on promoting it until we had an in-place update process? Apologies if this has not been communicated widely enough... (Or if I have misunderstood something.)
